styles/_use-cases.scss

```
@@ -991,4 +943,557 @@ html.light {
+}
+
+/* Project cards styles for payments page */
+.use-case-payments .payments-projects-grid {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 40px 40px; /* 40px horizontal, 48px vertical */
+  row-gap: 48px;
+
+  @media (min-width: 1200px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(1, 1fr);
+    gap: 40px;
+  }
+}
+
+.use-case-payments .payments-project-card {
+  min-height: 260px;
+  position: relative;
+  padding: 32px;
+  .project-description {
+    text-align: left; /* Changed from center to left */
+    
+    .first-word {
+      color: #FFF;
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 24px; /* 150% */
+    }
+    
+    .rest-text {
+      color: var(--XRPL-Primary-White, #FFF);
+      font-family: "Work Sans";
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 24px;
+    }
+  }
+
+  /* Add the top border gradient for payments page */
+  &::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 0.25rem;
+    border-top-left-radius: 0.5rem;
+    border-top-right-radius: 0.5rem;
+  }
+
+  /* Stablecoin images */
+  .project-logo {
+    margin-Bottom: 32px;
+    img.ripple-usd {
+      content: url('../img/uses/payments/rlusd.png');
+      width: 180px;
+      height: 50px;
+    }
+    
+    img.usdc {
+      content: url('../img/uses/payments/usdc.png');
+      width: 50px;
+      height: 50px;
+    }
+    
+    img.usdb {
+      content: url('../img/uses/payments/usdb.png');
+      width: 126px;
+      height: 50px;
+    }
+    
+    img.europ {
+      content: url('../img/uses/payments/eroup.png');
+      width: 147px;
+      height: 50px;
+    }
+    
+    img.xsgd {
+      content: url('../img/uses/payments/XSGD.png');
+      width: 50px;
+      height: 50px;
+    }
+    
+    img.audd {
+      content: url('../img/uses/payments/AUDD.png');
+      width: 50px;
+      height: 50px;
+    }
+  }
+
+  /* Alternating gradients for payments page */
+  &.odd::before {
+    background: linear-gradient(90deg, #D91AFF 26.41%, #1AA4FF 100.32%);
+  }
+
+  &.even::before {
+    background: linear-gradient(90deg, #4BB7FF -0.32%, #32E685 30.61%);
+  }
+}
+
+/* Battle-tested section styles */
+.use-case-payments .battle-tested-section {
+  h4.eyebrow {
+    font-size: 28px !important;
+  }
+  
+  .payments-project-card {
+    /* Override styles for battle-tested cards */
+    min-height: 384px;
+    display: flex;
+    flex-direction: column;
+    
+    .project-logo img {
+      /* Battle-tested company logos */
+      
+      &.coinpayments {
+        content: url('../img/uses/payments/coinpayments.png');
+        width: 99px;
+        height: 60px;
+      }
+      
+      &.ripple {
+        content: url('../img/uses/payments/ripple-white.png');
+        width: 100px;
+        height: 30px;
```

This image (both the light mode and dark mode versions) is only 26px tall so it gets unpleasantly stretched by this style.

```suggestion
        height: 26px;
```